### PR TITLE
Feature/ambiente-aluno: Implementado seleção do card do tutor, botão para agendar a aula e modal de confirmação de agendamento da aula.

### PR DESCRIPTION
**Pull request by PabloDamato**
CHANGES 📝

Update: 

- lesson-card-manager.component.html,
- lesson-card-manager.component.scss,
- lesson-card-manager.component.ts,
- class-request-form.component.ts,
- button.component.scss,
- button.component.ts,
- main.component.ts,
- main.component.html,
- ITutorPersonalIData.interface.ts,
- class-request.interface.ts,
- class-request-data.interface.ts,
- lesson-request.interface.ts,
- tutor-response.interface.ts,
- environmentButton.enum.ts,
- tutor-request.component.ts,
- lesson-request.service.ts,
- loading.service.ts

Create:

- student-data.interface.ts,
- selected-request.interface.ts,
- tutor-modal-data.interface.ts
 
CARDS 📚
[Escolha do Tutor (Ambiente Aluno)](https://trello.com/c/2rvPEko3)

NOTE 📝
Branch "feature/ambiente-aluno" -> "main"

DETAILS 📝
Environment demonstration:

https://github.com/user-attachments/assets/3b932e7c-50ab-49b3-890f-05251ecb217e

### CODE REVIEW

File: src/shared/components/class-request-form/class-request-form.component.ts [hunk 1]
```diff
@@ -382,8 +382,19 @@ export class ClassRequestFormComponent implements OnInit {
...
+    const month = String(date.getMonth() + 1).padStart(2, '0');
+    const day = String(date.getDate()).padStart(2, '0');
+
+    // Remove qualquer segundo que possa vir no time e garante o formato HH:mm
+    const formattedTime = time.split(':').slice(0, 2).join(':');
+
+    // Retorna no formato YYYY-MM-DDThh:mm
+    // return `${year}-${month}-${day}T${formattedTime}`;
+    const formattedDate = `${year}-${month}-${day}T${formattedTime}`;
+    console.log('Data formatada:', formattedDate); // Log 1
```
Comment (aLiviaMs) [inline src/shared/components/class-request-form/class-request-form.component.ts]: n esqueca de remover os debuggers, n é bom manter console.logs

File: src/shared/components/class-request-form/class-request-form.component.ts [hunk 2]
```diff
@@ -382,8 +382,19 @@ export class ClassRequestFormComponent implements OnInit {
    * @private
    */
   private formatSchedule(date: Date, time: string): string {
-    const formattedDate = formatDate(date, 'dd/MM/yyyy', 'en-US');
-    return `${formattedDate} às ${time}`;
+    // Formata a data para YYYY-MM-DD
+    const year = date.getFullYear();
```
Comment (aLiviaMs) [inline src/shared/components/class-request-form/class-request-form.component.ts]: Este é o unico lugar q precisa de transformacao da data? Se NÃO, abstraia numa utilFunction, path/utils/date-utils.ts e ent chame a util aqui

**Review by aLiviaMs — CHANGES_REQUESTED** [review]

File: src/shared/components/class-request-form/class-request-form.component.ts (hunk 2 above)
Comment (PabloDamato) [reply src/shared/components/class-request-form/class-request-form.component.ts]: Feito.

File: src/shared/components/class-request-form/class-request-form.component.ts (hunk 1 above)
Comment (PabloDamato) [reply src/shared/components/class-request-form/class-request-form.component.ts]: Feito.

**Review by aLiviaMs — APPROVED** [review]

**Pull request merged**